Cleaning up the Readme (#33)
SuperDARN/pydarn: README.md

![pydarn](docs/imgs/pydarn_logo.png)

[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0) [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/) [![GitHub version](https://badge.fury.io/gh/boennemann%2Fbadges.svg)](http://badge.fury.io/gh/boennemann%2Fbadges)

Python data visualization library for the Super Dual Auroral Radar Network (SuperDARN).

## Changelog

## Version 1.0 - Release!

> pyDarn is released! Included are the following features:

Reviewer: ```suggestion
pyDARN is released! Included are the following features:
```
I know this is mine, but just for consistency with the rest of the readme.

- Reading and writing DMap format IQDAT, RAWACF, FITACF, GRID/GRD and MAP files
- Reading and writing HDF5 format files for Borealis radar data, as well as conversion to and from DMap format
- Range-time parameter style plots for RAWACF and FITACF files
- Summary plots for RAWACF and FITACF files
- Time series plots for RAWACF and FITACF files

## Documentation

pyDARN's documentation can found [here](https://pydarn.readthedocs.io/en/latest)

## Getting Started

To install and use pyDARN please read the [installation guide](https://pydarn.readthedocs.io/en/latest/user/install/).

> If wish to get access to SuperDARN data please read [SuperDARN data access documentation](https://pydarn.readthedocs.io/en/latest/user/superdarn_data/).

Reviewer: ```suggestion
If wish to get access to SuperDARN data please read the [SuperDARN data access documentation](https://pydarn.readthedocs.io/en/latest/user/superdarn_data/).
```

Please make sure to also read the documentation on [**citing superDARN and pydarn**](https://pydarn.readthedocs.io/en/latest/user/citing/). 

As a quick tutorial on using pydarn to read a compressed file: 
``` python
import bz2
import pydarn 

# read in compressed file
fitacf_file = '20180220.C0.rkn.stream.fitacf.bz2'
with bz2.open(fitacf_file) as fp: 

>       fitacf_stream = fp.read()

Reviewer: I wouldn't mind opinions from others if this is the common way they open files? From my understanding, most people will uncompress the file before trying to read it. So these instructions might only apply for a minority.

Additionally, tutorials on reading in files can be found in the documentation, so this might be better suited there instead of clogging up the readme.

Author: I want to have one example to get people started just in case they are lazy about reading the documentation. We can switch it to a more general case if reading compressed files seems less common. That is why I said there is more in the documentation for tutorials to help lead users to it.

reader = pydarn.SDarnRead(fitacf_stream, True)
records = reader.read_fitacf()
```

For more information and tutorials on pyDARN please see the [tutorial section](https://pydarn.readthedocs.io/en/latest/)

## Getting involved

pyDARN is always looking for testers and developers keen on learning python, github, and/or SuperDARN data visualizations! 
Here are some ways to get started: 

  - **Testing Pull Request**: to determine which [pull requests](https://github.com/SuperDARN/pydarn/pulls) need to be tested right away, filter them by their milestones.

>   - **Getting involved in projects**: if you are looking to help in a specific area, look at pyDARN's [projects tab](https://github.com/SuperDARN/pydarn/projects). The project you are interested in will give you the information on what is needed to get done, in progress, and to be reviewed. 

Reviewer: ```suggestion
  - **Getting involved in projects**: if you are looking to help in a specific area, look at pyDARN's [projects tab](https://github.com/SuperDARN/pydarn/projects). The project you are interested in will give you information on what is needed to reach completion. This includes things currently in progress, and those awaiting reviews. 
```

>   - **Answer questions**: if you want to try your hand at answer some pyDARN questions or adding to the discussion, look at pyDARN's [issues](https://github.com/SuperDARN/pydarn/issues) and filter by labels.

Reviewer: ```suggestion
  - **Answer questions**: if you want to try your hand at answering some pyDARN questions, or adding to the discussion, look at pyDARN's [issues](https://github.com/SuperDARN/pydarn/issues) and filter by labels.
```

>   - **Become a developer**: if you want to practice those coding skills and at to the library, look at pyDARN [issues](https://github.com/SuperDARN/pydarn/issues) and filter by milestone's to see what needs to get done right away. 

Reviewer: ```suggestion
  - **Become a developer**: if you want to practice those coding skills and add to the library, look at pyDARN [issues](https://github.com/SuperDARN/pydarn/issues) and filter by milestone's to see what needs to get done right away. 
```

> Please contact the leading developer or group if you would like to become a member of the team!

Reviewer: Not sure what this means exactly. The person reading won't know who the leading developer or group is unless you put them down here :P

Author: Fixed
